src/index.ts

```diff
@@ -137,7 +137,7 @@ export { useBandwidthMetrics } from './hooks/sdk/useBandwidthMetrics';
 
 // Providers
 export { NotificationProvider } from './providers/NotificationProvider';
-export { MeetingProvider } from './providers/MeetingProvider';
+export { MeetingProvider, MeetingContext } from './providers/MeetingProvider';
```

Comment:
We have not exported the contexts, rather have the `use..` hooks exported that provide the context, so should we export all contexts or none?

Reply:
Meant to remove this, thanks -- I think there's a use case for exporting the contexts, but not in this PR